### ISSUE TITLE
OpenEMR video tutorials and work flows 

### ISSUE THREAD

**Issue by usama-kabel** [openemr/openemr#5771]
### Describe the problem

The issue is OpenEMRhas many many features but is not presented enough or how can use it in the better way
the existing videos have not been updated for a long time with old versions, I know the functionality may be its same in the new versions but at least we can show the updated things added in these parts.

also very important, the globals setting and how it's affects on OpenEMR, this part not covered in any place I think.

### Describe the solution you'd like

I think it's very important now to have a good video learning and explanation for the features and the best flow for this feature

**Comment by bradymiller** [comment]
yep, agree need more updated videos. The door is open for anybody to create these. The resources to create videos is relatively steep (for example, I am working on a video about the easy dev environment for OpenEMR and it takes a bit of time).

Regarding globals, we were doing a good job tracking that at this page in the past (however, it is about a year behind and needs to be updated):
https://www.open-emr.org/wiki/index.php/Administration_Globals

**Comment by usama-kabel** [comment]
We need to put the updated videos for each module
